controllers/map-filter.js

```diff
@@ -35,8 +35,19 @@ function config (req, res, next) {
       { name: 'Local', path: '/mapfilter/filters/local', value: 'local' }
     ],
     bingProxy: '/bing-proxy',
-    bingMetadata: '/bing-metadata'
+    bingMetadata: '/bing-metadata',
+    tiles: {
+      url: '/tileLayers',
+      tilesPath: '/monitoring-files/Maps/Tiles'
+    }
   };
+  if (exists(settings.getTracksDirectory())) {
```

Comment:
This essentially does what I mentioned above -- check first, only add if the folder exists.